packages/lit-html/src/directives/live.ts

```diff
@@ -33,31 +51,25 @@ class LiveDirective extends Directive {
     }
   }
 
-  render(value: unknown) {
+  render(value: unknown, _equal?: EqualFn) {
     return value;
   }
 
-  override update(part: AttributePart, [value]: DirectiveParameters<this>) {
+  override update(
+    part: AttributePart & {
+      readonly type: ValidPartType;
+    },
+    [value, equal]: DirectiveParameters<this>
```

Comment:
Defaults can be set in the destructuring parameter:

```suggestion
    [value, equal = defaults[part.type]]: DirectiveParameters<this>
```

Reply:
Updated.